rosetta/package.json

```diff
@@ -2,10 +2,18 @@
   "name": "rosetta",
   "version": "0.1.0",
   "devDependencies": {
+    "backbone": "^1.3.3",
+    "underscore": "^1.8.3",
+    "jquery": "^2.2.3",
+    "handlebars": "^4.0.5",
+    "bootstrap": "^3.3.6",
+    "normalize-css": "^2.3.1",
```

Comment:
Npm says that it is the highest version of which it has references. The message comes up doing npm install command. Performing  tests are normal with this lower version.